docs/AboutUs.md

```diff
@@ -41,7 +41,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ### HsiaoJietNg
 
-<img src="images/team/hsiaojietng.jpeg" width="200px">
+<img src="images/hsiaojietng.jpeg" width="200px">
```

Comment:
File extension should be `.png` as described [here](https://nus-cs2103-ay2122s2.github.io/website/schedule/week7/project.html#2-update-project-website-aboutus-readme).